python/ray/serve/tests/conftest.py

```diff
@@ -151,6 +151,28 @@ def ray_start_stop():
     )
 
 
+@pytest.fixture(scope="function")
+def ray_start_stop_in_specific_directory(request):
+    # Change working directory so Ray will start in the correct directory.
+    working_dir = request.param
+    os.chdir(working_dir)
+
+    subprocess.check_output(["ray", "start", "--head"])
+    wait_for_condition(
+        lambda: requests.get("http://localhost:52365/api/ray/version").status_code
+        == 200,
+        timeout=15,
+    )
+    try:
+        yield
+    finally:
+        subprocess.check_output(["ray", "stop", "--force"])
+        wait_for_condition(
+            check_ray_stop,
+            timeout=15,
+        )
```

Comment:
Change the directory back, so tests after this one don't also start in this directory.

Reply:
good call! Thanks!